core/build/headers/sgx_error.h

```diff
@@ -75,6 +75,7 @@ typedef enum _status_t
     SGX_ERROR_INVALID_CPUSVN     = SGX_MK_ERROR(0x3003),      /* The cpu svn is beyond platform's cpu svn value */
     SGX_ERROR_INVALID_ISVSVN     = SGX_MK_ERROR(0x3004),      /* The isv svn is greater than the enclave's isv svn */
     SGX_ERROR_INVALID_KEYNAME    = SGX_MK_ERROR(0x3005),      /* The key name is an unsupported value */
```

Comment:
new addition in [sgx_error.h](https://github.com/intel/linux-sgx/blob/sgx_2.25_reproducible/common/inc/sgx_error.h)